test/integration/acceptance/set_account_quorum_test.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include "framework/integration_framework/integration_test_framework.hpp"
+#include "integration/acceptance/acceptance_fixture.hpp"
+
+using namespace integration_framework;
+using namespace shared_model;
+using namespace common_constants;
+
+#define check(i) [](auto &block) { ASSERT_EQ(block->transactions().size(), i); }
```

Comment:
can we move that macro to acceptance fixture, since it is used in many tests?

Reply:
Let's do it as a separate pull request (IR-1869).